src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -151,138 +175,71 @@ private void generateStatusMessage(AdminActivityLogPageData data, List<ActivityL
         statusToUser.add(new StatusMessage(status, StatusMessageColor.INFO));
     }
 
+    /**
+     * Retrieves enough logs within 24 hour.
+     */
```

Comment:
That way you don't have to hard-code 24 hours here. You can make it refer to the variable that stores the 24 hours (or whatever value it is, if going to be changed).